ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -327,4 +346,36 @@ private static boolean bug1733118Affected(String namespace, String docType, Obje
             .matches("0[.]([0-9]|1[0-6])[.].*"); // < 0.17
   }
 
+  // See bug 1751753 for explanation of context.
+  private static void processForBug1751753(ObjectNode json) {
+    processKeysForBug1751753(json.path("payload").path("keyedHistograms").path("SEARCH_COUNTS"));
+    json.path("payload").path("processes").path("parent").path("keyedScalars").fields()
+        .forEachRemaining(entry -> {
+          if (entry.getKey().startsWith("browser.search.content.")) {
+            processKeysForBug1751753(entry.getValue());
+          }
+        });
+  }
+
+  private static void processKeysForBug1751753(JsonNode inputNode) {
+    if (!inputNode.isObject()) {
+      // If this isn't an object, we have nothing to do here.
+      return;
+    }
+    ObjectNode searchNode = (ObjectNode) inputNode;
+    searchNode.fields().forEachRemaining(entry -> {
```

Comment:
I'll need to recast this as currently it raises ConcurrentModificationException in some cases.